core/src/commonMain/kotlin/com/ustadmobile/core/domain/share/SendAppFileUseCase.kt

```diff
@@ -0,0 +1,5 @@
+package com.ustadmobile.core.domain.share
+
+interface SendAppFileUseCase {
```

Comment:
Let's rename this to ShareAppUseCase because it is now having both the send file option and share app link.